content/en/blog/2022/SIG-Storage Spotlight.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: blog
+title: "SIG-Storage Spotlight"
```

Comment:
How about aiming for publication on the 2nd of August?
(we can move it earlier if editing is done by then).
```suggestion
title: "Spotlight on SIG Storage"
date: 2022-08-02
```

For the title, compare https://www.k8s.dev/blog/2022/02/04/sig-multicluster-spotlight-2022/

Reply:
Publication date seems fine; I used https://kubernetes.io/blog/2020/06/30/sig-windows-spotlight-2020/ as a reference, hence the different approach.